miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/ListPoolsController.java

```diff
@@ -74,7 +74,12 @@ public void setRequestManager(RequestManager requestManager) {
 
   @ModelAttribute("platformTypes")
   public Collection<String> populatePlatformTypes() throws IOException {
-    return PlatformType.platformTypeNames(requestManager.listActivePlatformTypes());
+    Collection<PlatformType> platforms = requestManager.listActivePlatformTypes();
+    if (platforms.size() > 0) {
+      return PlatformType.platformTypeNames(requestManager.listActivePlatformTypes());
```

Comment:
Can `return PlatformType.platformTypeNames(platforms);` instead of doing the same lookup twice